content/desktop/containerd.md

```diff
@@ -8,173 +8,76 @@ toc_max: 3
 This page provides information about the ongoing integration of `containerd` for
 image and file system management in the Docker Engine.
 
-## What is the containerd image store?
-
-`containerd` is a container runtime that manages the container lifecycle, and
-provides image and filesystem management. It's a low-level building block,
-designed to be integrated into other systems, such as Docker and Kubernetes.
-
-Docker Engine already uses `containerd` for container lifecycle management, which
-includes creating, starting, and stopping containers. This page describes the
-next step of containerd integration for Docker Engine: the image store.
-
-The image store is the component responsible for pushing, pulling, and storing
-images. Integrating the containerd image store enables many new features in the
-Docker Engine, including:
-
-- containerd snapshotters, such as [stargz][1] for lazy-pulling images on startup,
+## What is containerd?
+
+`containerd` is an abstraction of the low-level kernel features
+used to run and manage containers on a system.
+It's a platform used in container software like Docker and Kubernetes,
```

Comment:
The comma after Kubernetes reads rough. Maybe use a period or semicolon. Or, some conjunction after the comma.

Reply:
Oops, that was supposed to be a `.` - fixing